src/Adapter/Admin/AbstractAdminQueryBuilder.php

```diff
@@ -117,7 +117,7 @@ private function compileSqlWhere(array $whereArray)
      * Format example for $order:
      * $order = array('name ASC', 'id_product DESC');
      *
-     * @param array<array<mixed>> $select
+     * @param array<string,array<string,string>|string> $select
```

Comment:
PHPStan is able to detect if someone calls `compileSqlQuery()` with a bad `$select` input (like an array of integers) ?

Reply:
No, I have to reverse engineering use of `compileSqlQuery`.